src/common/FxPresetAndClipboardManager.cpp

```diff
@@ -195,35 +195,34 @@ bool FxUserPreset::readFromXMLSnapshot(Preset &preset, TiXmlElement *s)
     {
         double fl;
         std::string p = "p";
+        std::string tmp; // Workaround for a g++12 bug
 
-        if (s->QueryDoubleAttribute((p + std::to_string(i)).c_str(), &fl) == TIXML_SUCCESS)
+        tmp = p + std::to_string(i);
+        if (s->QueryDoubleAttribute(p.c_str(), &fl) == TIXML_SUCCESS)
```

Comment:
This should be tmp

Reply:
Yeah; this fix didn't work anyway...resolving comment since I changed the fix.